### PR TITLE
update CHANGELOG.md for SE-266

### PR DESCRIPTION
**Pull request by tayloraswift**
title says it all

### CODE REVIEW

**Comment by CodaFi** [comment]
Will squash this down manually

@swift-ci please smoke test

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,18 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-266](https://github.com/apple/swift-evolution/blob/master/proposals/0266-synthesized-comparable-for-enumerations.md):
```
Comment (theblixguy) [inline CHANGELOG.md]: The URL should be put at the bottom I think...

Comment (kelvin13) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,18 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-266][]:
```
Comment (theblixguy) [inline CHANGELOG.md]:
```suggestion
* [SE-0266][]:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7901,6 +7913,7 @@ Swift 1.0
 [SE-0252]: <https://github.com/apple/swift-evolution/blob/master/proposals/0252-keypath-dynamic-member-lookup.md>
 [SE-0253]: <https://github.com/apple/swift-evolution/blob/master/proposals/0253-callable.md>
 [SE-0254]: <https://github.com/apple/swift-evolution/blob/master/proposals/0254-static-subscripts.md>
+[SE-266]: <https://github.com/apple/swift-evolution/blob/master/proposals/0266-synthesized-comparable-for-enumerations.md>
```
Comment (theblixguy) [inline CHANGELOG.md]:
```suggestion
[SE-0266]: <https://github.com/apple/swift-evolution/blob/master/proposals/0266-synthesized-comparable-for-enumerations.md>
```

**Comment by theblixguy** [comment]
@swift-ci please smoke test

**Pull request merged**